spec/lib/danger/danger_core/plugins/dangerfile_git_plugin_spec.rb

```diff
@@ -99,6 +99,27 @@ module Danger
           end
         end
       end
+
+      describe "getting stats for a specific file" do
+        it "returns nil when specific stats do not exist" do
+          run_in_repo_with_diff do |git|
+            diff = git.diff("master")
+            allow(@repo).to receive(:diff).and_return(diff)
```

Comment:
Good call, that was cargo-cult code. Now that I'm adding file contents I probably will end up stubbing though. Thanks for the sharp eyes.